server/ERNI.PBA.Server.Host/Controllers/BudgetController.cs

```diff
@@ -51,16 +51,22 @@ public async Task<IActionResult> GetUserBudgetByYear(int userId, int year, Cance
         }
 
         [HttpGet("{budgetId}")]
-        [Authorize(Roles = Roles.Admin + "," + Roles.Viewer)]
-        public async Task<IActionResult> GetUserBudgetByYear(int budgetId, CancellationToken cancellationToken)
+        [Authorize]
```

Comment:
Maybe I asked this question already, but what about the viewer role?

Reply:
This endpoint is called only from otherBudgetsDetail.component. Thats route is admin only.  I dont think that check for viewer is needed